test/image/mocks/heatmap_hoverlabel.json

```diff
@@ -0,0 +1,39 @@
+{
+  "data": [
```

Comment:
A new mock is _ok_, but we should really test the new behavior under 

- https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/heatmap_test.js#L592
- https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/hover_label_test.js#L1238